### PR TITLE
Changing the title of the Aspire starter project

### PR DESCRIPTION
**Pull request by sayedihashimi**
Changing the title of the Aspire starter project template from '.NET Aspire Starter Application' to '.NET Aspire Starter App'.
This is more consistent with Visual Studio.
Related issue #4178.

Let me know if this PR should go to a different branch.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/4207)

### CODE REVIEW

**Comment by DamianEdwards** [comment]
@sayedihashimi can you update the empty template too? https://github.com/dotnet/aspire/blob/21a58881c899b7c4aee75aa24508e2adefeabd2f/src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json#L13

**Comment by sayedihashimi** [comment]
> @sayedihashimi can you update the empty template too?
> 
> https://github.com/dotnet/aspire/blob/21a58881c899b7c4aee75aa24508e2adefeabd2f/src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json#L13
> 
> Maybe we should probably rename that one to ".NET Aspire Empty App" while we're here

Sorry I missed that one when I made the initial PR, it has been updated now.

File: src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "API",
     "Service"
   ],
-  "name": ".NET Aspire Application",
+  "name": ".NET Aspire App",
```
Comment (DamianEdwards) [inline src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json:13]: Let's make it ".NET Aspire Empty App"

Comment (sayedihashimi) [reply src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json]: Done

**Review by DamianEdwards — APPROVED** [review]
Thanks!

**Pull request merged**